pypeit/spectrographs/magellan_fire.py

```diff
@@ -40,60 +118,51 @@ def __init__(self):
                             xgap            = 0.,
                             ygap            = 0.,
                             ysize           = 1.,
-                            platescale      = 0.15,
+                            platescale      = 0.18,
                             darkcurr        = 0.01,
-                            saturation      = 20000., # high gain mode, low gain is 32000
+                            saturation      = 20000., # high gain is 20000., low gain is 32000
```

Comment:
I think we may be using counts now instead of ADU. I'm worried this is too agressive. Maybe we should make this 100,000 and let user beware.

Reply:
I just made it to 100,000